app/controllers/books_controller.rb

```diff
@@ -0,0 +1,46 @@
+class BooksController < ApplicationController
+  def index 
+	  @books=Book.all
+  end
+
+  def show
+    @book = Book.find(params[:id])
```

Comment:
Try before_action or before_filter to refactor this line for show, edit and destroy.